### DISCUSSION TITLE
Microsoft Store forced update of Windows Terminal, killing busy Powershell instance

### DISCUSSION THREAD

**Discussion by southisup** [microsoft/terminal#13845] [General]
Not thinking, I did a Get Updates in Microsoft Store Library this morning and without warning it forced an update of Windows Terminal, killing a contained Powershell instance in which a scoop update was running, resulting in a broken main bucket. Nothing major in the end, but raising it here on the off chance someone holds sway over the Microsoft Store team.

**Comment by jessey-git** [comment]
Yeah, the same happens when awful IT departments force store auto-updates to be turned on.  All windows gone.  I can't believe Terminal will be the default in win11 with this behavior in place -- unless it's somehow better handled there than on win10(specifically with auto-update enabled).

**Comment by zadjii-msft** [comment]
FWIW This has been discussed to death in #3915. Talks with the Store team are still ongoing. There's not much more for us to do at this point - bothering the Store team any more often would only distract them from actually just doing the work.

**Reply by southisup** [reply]
Many thanks for the link, I couldn't find an open issue and feel it definitely deserves one. Happy for mine to be locked/closed now.
